### PR TITLE
example to echo out put from previous step fails

### PR DESCRIPTION
**Pull request by bcbrian**
### Why: 
I copied and pasted this piece out of the docs and it did not work.
The error is a bash error:  bad substitution.

[Link to the change](https://docs-528--patch-1.herokuapp.com/en/free-pro-team@latest/actions/reference/workflow-commands-for-github-actions#using-workflow-commands-to-access-toolkit-functions)

### What's being changed:
change - ```run: echo 'The selected color is' ${steps.random-color-generator.outputs.SELECTED_COLOR}```
to - ```run: echo "The selected color is ${{ steps.random-color-generator.outputs.SELECTED_COLOR }}"```

Other parts of the docs have it formatted this way. 

### Check off the following:
- [ ] All of the tests are passing.
- [x] I have reviewed my changes in staging.
- [x] For content changes, I have reviewed the [localization checklist](https://github.com/github/docs/blob/main/contributing/localization-checklist.md)
- [x] For content changes, I have reviewed the [Content style guide for GitHub Docs](https://github.com/github/docs/blob/main/contributing/content-style-guide.md).

### CODE REVIEW

File: content/actions/reference/workflow-commands-for-github-actions.md [hunk 1]
```diff
@@ -58,7 +58,7 @@ You can use the `set-output` command in your workflow to set the same value:
         run: echo '::set-output name=SELECTED_COLOR::green'
         id: random-color-generator
       - name: Get color
-        run: echo 'The selected color is' ${steps.random-color-generator.outputs.SELECTED_COLOR}
+        run: echo "The selected color is ${{ steps.random-color-generator.outputs.SELECTED_COLOR }}"
```
Comment (bcbrian) [inline content/actions/reference/workflow-commands-for-github-actions.md:61]: The double curly brackets need to be escaped...
the output is currently
```
run: echo "The selected color is $"
```

File: content/actions/reference/workflow-commands-for-github-actions.md [hunk 2]
```diff
@@ -53,13 +53,16 @@ core.setOutput('SELECTED_COLOR', 'green');
...
 ``` yaml
       - name: Set selected color
         run: echo '::set-output name=SELECTED_COLOR::green'
         id: random-color-generator
       - name: Get color
-        run: echo 'The selected color is' ${steps.random-color-generator.outputs.SELECTED_COLOR}
+        run: echo "The selected color is ${{ steps.random-color-generator.outputs.SELECTED_COLOR }}"
 ```
+{% endraw %}
+
```
Comment (bcbrian) [inline content/actions/reference/workflow-commands-for-github-actions.md:65]:
```suggestion
```

File: content/actions/reference/workflow-commands-for-github-actions.md (hunk 1 above)
Comment (bcbrian) [reply content/actions/reference/workflow-commands-for-github-actions.md:62]: raw fixed it

**Comment by janiceilene** [comment]
Thanks so much for opening a PR @bcbrian! I'll get this triaged and up for review ⚡

File: content/actions/reference/workflow-commands-for-github-actions.md (hunk 1 above)
Comment (lucascosti) [reply content/actions/reference/workflow-commands-for-github-actions.md:62]: Great work fixing that! Yep, double curly brackets are processed as variables etc in Liquid, so literal ones need to be escaped with `{% raw %}` 👍

**Review by lucascosti — APPROVED** [review]
Great stuff, @bcbrian! 🚀 

Thanks for fixing this, I'll get it merged when the tests from the rebase pass 👍

File: content/actions/reference/workflow-commands-for-github-actions.md (hunk 1 above)
Comment (mnznz) [reply content/actions/reference/workflow-commands-for-github-actions.md:62]: #582 #491 #358 @technicalpickles @myobie @mtodd @zmoazeni

**Review by mnznz — COMMENTED** [review]
@mnznz

**Pull request merged**